tests/integration/targets/gitlab_project_members/aliases

```diff
@@ -1 +1,5 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
 unsupported
```

Comment:
We might as well fix this (no new line at end of file) while we're already touching the file...

Reply:
Let's better do this in a follow-up PR. There are hundrets of files that have this, and some might already be touched in this PR while others are not.

Reply:
https://stackoverflow.com/a/44747386 seems to be helpful to do this for a large number of files at once.